src/local-settings.js

```diff
@@ -60,6 +61,7 @@ const CONFIGURATION_VARS = [
   [LOCAL_CHAIN_ID, process.env.LOCAL_CHAIN_ID],
   [FORTMATIC_API_KEY, process.env.ARAGON_FORTMATIC_API_KEY],
   [PORTIS_DAPP_ID, process.env.ARAGON_PORTIS_DAPP_ID],
+  [WALLET_CONNECT_RPC, process.env.ARAGON_WALLET_CONNECT_RPC],
```

Comment:
Add `getWalletConnectRpc()` helper below
This env variable is useless right now

Also add it in `.env.example`